specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/Updates.json

```diff
@@ -0,0 +1,475 @@
+        },
+        "nextLink": {
+          "description": "URI to the next page of updates.",
+          "type": "string"
+        }
+      }
+    },
+    "UpdateState": {
+      "description": "State of the update.",
+      "type": "string",
+      "enum": [
+        "Available",
+        "Downloading",
+        "DownloadFailed",
+        "HealthChecking",
+        "HealthCheckFailed",
+        "Preparing",
+        "PreparationFailed",
+        "Invalid",
+        "HasPrerequisite",
+        "Recalled",
+        "Obsolete",
+        "ReadyToInstall",
+        "NotApplicableBecauseAnotherUpdateIsInProgress",
+        "Installing",
+        "Installed",
+        "AttentionNeeded",
+        "InstallationFailed"
+      ],
+      "x-ms-enum": {
+        "name": "UpdateState",
+        "modelAsString": true
+      }
+    },
+    "AvailabilityType": {
+      "description": "Availability type of update.",
+      "type": "string",
+      "enum": [
+        "online",
+        "local",
+        "notify"
+      ],
+      "x-ms-enum": {
+        "name": "AvailabilityType",
+        "modelAsString": true
+      }
+    },
+    "Prerequisite": {
+      "description": "Prerequisite of the update.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "Title of the prerequisite.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the prerequisite.",
+          "type": "string"
+        },
+        "informationLink": {
+          "description": "Link providing details about prerequisite.",
+          "type": "string"
+        }
+      }
+    },
+    "UpdateStateProperties": {
+      "description": "State related properties of the update.",
+      "type": "object",
+      "properties": {
+        "progressPercentage": {
+          "description": "Overall percentage of a running sate.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "installedDate": {
+          "description": "Date update was installed.",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "AzsUpdateProperties": {
+      "description": "Properties for Microsoft update.",
+      "type": "object",
+      "properties": {
+        "minVersionRequired": {
+          "description": "The minimum version required to install.",
+          "type": "string"
+        },
+        "minOemVersionRequired": {
+          "description": "The minimum OEM version required to install.",
+          "type": "string"
+        }
+      }
+    },
+    "OemUpdateProperties": {
+      "description": "Properties for OEM update.",
+      "type": "object",
+      "properties": {
+        "oemFamily": {
+          "description": "The family of models to which the OEM update is applicable.",
+          "type": "string"
+        },
+        "oemNotifyMsg": {
+          "description": "The message from the OEM availability a relevant OEM update.",
```

Comment:
Maybe a typo or miss copy-paste? This sentence seems incomplete.

Reply:
It should be "The message from the OEM availability of a relevant OEM update.". The fix is on my local.